pkg/agent/cniserver/ipam/antrea_ipam_controller.go

```diff
@@ -208,7 +209,7 @@ func (c *AntreaIPAMController) getPoolAllocatorByPod(namespace, podName string)
 			}
 			klog.InfoS("IPPool not found", "pool", p)
 			err = nil
-		} else if allocator.IPVersion == crdv1a2.IPv4 {
+		} else if allocator.IPVersion == utilip.IPv4 {
```

Comment:
We can use `k8s.io/api/core/v1.IPFamily`, no need to define new constants.

Reply:
Second thought: it could use `"k8s.io/utils/net.IPFamily"`, then we can use util functions like `IPFamilyOf`, `IPFamilyOfString`.

Reply:
Done.